types/ocpp.yaml

```diff
@@ -451,13 +484,42 @@ types:
     properties:
       reason_code:
         description: >-
-          A predefined code for the reason why the status is returned in this response. 
+          A predefined code for the reason why the status is returned in this response.
           The string is case-insensitive.
         type: string
       additional_info:
         description: >-
           Additional text to provide detailed information
         type: string
+  TransactionEvent:
+    description: >-
+      Element providing information transactions.
+    type: object
+    required:
+      - event
+      - connector
+      - session_id
+    properties:
+      event:
+        description: >-
+          The transaction related event.
+        type: string
+        enum:
+          - Start
+          - Update
+          - End
+      connector:
```

Comment:
This should probably also include the evse since it's being used in OCPP 2.0.1 as well

Reply:
The evse id is included as an optional element in types/ocpp.yaml
perhaps there has been a merge issue?